Classes/ExplorerInterface/FLEXExplorerViewController.m

```diff
@@ -102,7 +102,7 @@ - (void)viewDidLoad
     CGFloat toolbarOriginY = toolbarOriginYDefault ? [toolbarOriginYDefault doubleValue] : 100;
 
     CGRect safeArea = [self viewSafeArea];
-    CGSize toolbarSize = [self.explorerToolbar sizeThatFits:CGSizeMake(CGRectGetWidth(self.view.bounds), CGRectGetHeight(safeArea))];
+    CGSize toolbarSize = [self.explorerToolbar sizeThatFits:CGSizeMake(70, CGRectGetHeight(safeArea))];
```

Comment:
Just a headsup/ question:
The 70 value, is it pixels? Will this behave properly in higher density screens?

Reply:
I tested with multiple devices and it worked properly in all of them. The handle to drag the view also has its width defined with a constant that is used directly in the frame.